cli/yara.c

```diff
@@ -1197,6 +1197,10 @@ static int callback(
       return CALLBACK_ERROR;
 
     return CALLBACK_CONTINUE;
+
+  case CALLBACK_MSG_CONSOLE_LOG:
+    printf("%s", (char*) message_data);
```

Comment:
The library user code (i.e. yara(1)) should be responsible for inserting newlines or whatever separator it wants to add.

It might be a good idea to add a prefix here.

Reply:
Good point! Thanks. :)